src/NuGet.Services.Search/NuGet.Services.Search.csproj

```diff
@@ -188,7 +188,7 @@
       <HintPath>..\..\packages\NuGet.Core.2.8.5\lib\net40-Client\NuGet.Core.dll</HintPath>
     </Reference>
     <Reference Include="NuGet.Indexing, Version=0.0.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\NuGet.Indexing.3.0.280-r-master\lib\net45\NuGet.Indexing.dll</HintPath>
```

Comment:
We should stick with 3.0.280 version for NuGet.Indexing. The latest version has indexing changes that are done as part of v2/v3 merge and it may not work with v2 search.